src/javascript/grid3_javascript_qsfs_zdbs.md

```diff
@@ -0,0 +1,123 @@
+first, make sure you have your [client](./grid3_javascript_loadclient.md) prepared
+
+## Deploying ZDBs for QSFS
+
+### Example code
+
+````typescript
+import { FilterOptions, QSFSZDBSModel } from "../src";
+import { getClient } from "./client_loader";
+import { log } from "./utils";
+
+async function main() {
+    const grid3 = await getClient();
+    const qsfs_name = "zdbsQsfsDemo";
+    const qsfsQueryOptions: FilterOptions = {
+        hru: 16,
+        availableFor: grid3.twinId,
+        farmId: 1,
+    };
+    const qsfsNodes = [];
+
+    const allNodes = await grid3.capacity.filterNodes(qsfsQueryOptions);
+
+    if (allNodes.length >= 2) {
+        qsfsNodes.push(+allNodes[0].nodeId, +allNodes[1].nodeId);
+    } else {
+        throw Error("Couldn't find nodes for qsfs");
+    }
+
+    const qsfs: QSFSZDBSModel = {
+        name: qsfs_name,
+        count: 12,
+        node_ids: qsfsNodes,
+        password: "mypassword",
+        disk_size: 1,
+        description: "my zdbs test",
+        metadata: "",
+    };
+    const deploy_res = await grid3.qsfs_zdbs.deploy(qsfs);
+    log(deploy_res);
+
+    const zdbs_data = await grid3.qsfs_zdbs.get({ name: qsfs_name });
+    log(zdbs_data);
+
+    const delete_response = await grid3.qsfs_zdbs.delete({ name: qsfs_name });
+    log(delete_response);
```

Comment:
let's make the deletion part commented to not deploy and delete at the same time

Reply:
don't comment please, just remove it